### PR TITLE
Improve stacktrace format and clean main.cc

### PR DESCRIPTION
**Pull request by PragmaTwice**
- Align function names in stacktrace (in `segvHandler`)
- Remove many useless code

### CODE REVIEW

**Review by tisonkun — COMMENTED** [review]
Any description (what is to be implemented) of this change? It seems miscellaneous in one patch.

**Comment by PragmaTwice** [comment]
> Any description (what is to be implemented) of this change? It seems miscellaneous in one patch.

Added.

File: src/main.cc [hunk 1]
```diff
@@ -281,8 +290,8 @@ int main(int argc, char *argv[]) {
   evthread_use_pthreads();
 
   signal(SIGPIPE, SIG_IGN);
-  signal(SIGINT, signal_handler);
-  signal(SIGTERM, signal_handler);
+  signal(SIGINT, signalHandler);
+  signal(SIGTERM, signalHandler);
```
Comment (tisonkun) [inline src/main.cc:294]: I'm unsure whether this handle can be still registered properly. To me, the original `extern "C"` block is for C ABI compatibility.

Comment (PragmaTwice) [reply src/main.cc:294]: I re-added them.

**Review by git-hulk — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Pull request merged**